app/middleware/getGp.js

```diff
@@ -22,7 +22,7 @@ function getGp(req, res, next) {
       next();
     });
   }).catch((err) => {
-    log.error(err.stack, `Error connecting to ${connectionString}`);
+    log.error(err, `Error connecting to ${connectionString}`);
```

Comment:
I'm going to bet that is because of backwards compatibility with node envs where string interpolation isn't supported.